src/components/Advertisements/core/AdvertisementRegister/AdvertisementRegisterMocks.ts

```
@@ -0,0 +1,193 @@
+      },
+    },
+  );
+};
+
+const createAdvertisementNode = (
+  id: string,
+  name: string,
+  description: string,
+  startAt: string,
+  endAt: string,
+  type: string = 'banner',
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)